RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+  ![Booking a receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument01.png "[Booking a receipt]")
+
+7. Click the [BOOK] button.  
+A message above the entry line confirms that the receipt has been linked to the booking. A new booking is displayed in the bookings list in the *BOOKING* tab. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the booking in the bookings list.
+
+  ![Receipt linked](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/DocumentLinked01.png "[Receipt linked]")
+
+  > [Info] If there are further unregistered external receipts, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the message "No documents left to be registered" is displayed.
+
+
+## Book a supplier receipt
+
+The *Supplier receipts* menu entry of the *EDIT* context menu in the *BOOKINGS* is an enhanced function to book specifically supplier receipts.
+
+Suppliers' invoices can be uploaded in PDF format in the *New supplier receipts* folder in the *Documents* module, which will be then recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once they are booked, they are moved automatically into the *Acquired documents* folder.
+
+ Moreover, this function interacts with the *Purchasing* and *Payment processing* modules, and allows to check supplier receipts before booking them in the system and to automatize payment conditions.
+
+[comment]: <> (ACQUIRE probably "enter" -ERFASSEN- or "park" -vorerfassen", SAP Terminology)
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- The appropriate plug-in is installed (Checking of supplier receipts + payment proposal list).
+- The corresponding folder is created in the *Documents* module.
+- Supplier receipts are uploaded in the *New supplier receipts* folder.
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Supplier receipts*
+
+![Supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceipts.png "[Supplier receipts]")
+
+1.  Click the *Supplier receipt* menu entry.  
+The *Supplier receipts* window is displayed. The external receipts uploaded in the *New supplier receipts* folder in the *Documents* module are displayed in the *Acquire* tab.
+
+    ![Acquire supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsAcquire.png "[Acquire supplier receipts]")
+
+    > [Info] If there are no receipts to be booked, the message *No documents left to be registered* is displayed.
+
+  [comment]: <> (They do not appear in Acquire if located in New supplier receipts folder, but in the New documents. Check why/if correct/if it is a bug?)
```

Comment:
Ich bekomme nie Dokumente angezeigt, obwohl ich sowohl im Neue Dokumente Ordner, als auch im Neue Lieferantenbelege Ordner Dokumente liegen habe. Was mache ich falsch?

Reply:
Das war bei mir genauso. Heute hat es aber funktioniert, nachdem ich den Supplier receipts Ordner erstellt habe. Ich muss evtl. Flo oder Nseim nochmals fragen, aber ich meine, so sollte es funktioniert, wie jetzt beschrieben. Evtl. in die Einstellungen des Documents Moduls gucken, ob die Pfade stimmen?